build.gradle

```diff
@@ -21,10 +21,19 @@ allprojects {
         jcenter()
         mavenCentral()
     }
+}
+
+task lint(dependsOn: ['spotlessApply'])
```

Comment:
Is it just `lint` => `spotlessCheck`? I'm not sure I've seen lint refer to a fixing step before.

Reply:
Thanks, this is term misusage on my end.  I've associated linting w/ the fix features of eslint and flake8.  Corrected for `lint` to be the check and `format` for spotlessApply.